src/test/java/core/basesyntax/service/impl/DataParserServiceImplTest.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.Collections;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class DataParserServiceImplTest {
+    private DataParserServiceImpl dataParserService;
+
+    @BeforeEach
```

Comment:
Move the creation of service to the method with annotation `@BeforeAll`  the service doesn't contain the state and is not changed in different tests.
Fix all similar cases in all test classes where needed.